src/share/poudriere/common.sh

```diff
@@ -7477,6 +7477,24 @@ calculate_ospart_size() {
 	msg "OS Partiton size: ${OS_SIZE}m"
 }
 
+ingest_base_pkgs()
+{
+	# Do we have any base-port packages to ingest?
+	if [ ! -d "${MASTERMNT}/.packages/" ] ; then
+		return 0
+	fi
+
+	if [ ! -d "${PACKAGES}/All/" ] ; then
+		mkdir -p "${PACKAGES}/All/"
+	fi
+
+	for i in $(ls ${MASTERMNT}/.packages/*.txz)
```

Comment:
No reason to execute `ls`. You can avoid empty dir with a `case` as done elsewhere.